<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Реестр 223-ФЗ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/main.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tableRegisterView">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>29</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Экспорт в Excel</string>
     </property>
     <addaction name="normalExportAction"/>
     <addaction name="extendedExportAction"/>
    </widget>
    <addaction name="settingAction"/>
    <addaction name="reportAction"/>
    <addaction name="menu_6"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="informationAction"/>
    <addaction name="deleteAction"/>
    <addaction name="addAction"/>
    <addaction name="findAction"/>
    <addaction name="showRegisterAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Переключить реестр</string>
    </property>
    <addaction name="registerContractsAction"/>
    <addaction name="registerChecksAction"/>
    <addaction name="registerProcurementMethodsAction"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Справочники</string>
    </property>
    <addaction name="providersHandBookAction"/>
    <addaction name="subdivisionsHandBookAction"/>
    <addaction name="responsiblesHandBookAction"/>
    <addaction name="sourcesFinancingHandBookAction"/>
    <addaction name="paymentDocumentsHandBookAction"/>
    <addaction name="procurementMethodsHandBookAction"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addAction"/>
   <addaction name="informationAction"/>
   <addaction name="deleteAction"/>
   <addaction name="findAction"/>
   <addaction name="showRegisterAction"/>
   <addaction name="separator"/>
   <addaction name="registerContractsAction"/>
   <addaction name="registerChecksAction"/>
   <addaction name="registerProcurementMethodsAction"/>
   <addaction name="separator"/>
  </widget>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="informationAction">
   <property name="text">
    <string>Сведения</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="registerContractsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Реестр Договоров</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="registerChecksAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Реестр Счетов</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="registerProcurementMethodsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Реестр Конкурентных Способов Закупки</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="addAction">
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="providersHandBookAction">
   <property name="text">
    <string>Поставщики товаров/услуг</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="subdivisionsHandBookAction">
   <property name="text">
    <string>Подразделение-заказчик</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="responsiblesHandBookAction">
   <property name="text">
    <string>Ответственные лица</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="sourcesFinancingHandBookAction">
   <property name="text">
    <string>Источники финансирования</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="paymentDocumentsHandBookAction">
   <property name="text">
    <string>Платежные документы</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="procurementMethodsHandBookAction">
   <property name="text">
    <string>Способы закупки</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+6</string>
   </property>
  </action>
  <action name="findAction">
   <property name="text">
    <string>Поиск</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="showRegisterAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Показать реестр</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="normalExportAction">
   <property name="text">
    <string>Обычный</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="extendedExportAction">
   <property name="text">
    <string>Расширенный</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="settingAction">
   <property name="text">
    <string>Настройки</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="reportAction">
   <property name="text">
    <string>Отчеты</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F11</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tableRegisterView</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/main.qrc"/>
 </resources>
 <connections/>
</ui>
